src/hooks/useSwapCurrencyList.ts

```diff
@@ -409,7 +462,7 @@ const useSwapCurrencyList = (
             data: curatedAssets,
             key: 'curated',
             title: tokenSectionTypes.verifiedTokenSection,
-            useGradientText: IS_TESTING === 'true' ? false : true,
+            useGradientText: IS_TESTING !== 'true',
```

Comment:
i believe you can use IS_TEST now

Reply:
Alright. I think this diff was generated by the formatter, but I changed these lines to use IS_TEST instead.